krux_marathon_api/marathonapi.py

```diff
@@ -83,6 +68,10 @@ def read_config_file(self, config_file):
         return data
 
     def assign_config_data(self, config_file_data, marathon_app_result):
+        """
+        Assign variables from the config file to the marathon app values.
+        We cannot use a dymnamic variable so we are assigning these one at a time.
+        """
         ### assigning values from json to our marathon app
         self.logger.info("Assigning json variables to app data")
         marathon_app_result.cpus                     = config_file_data["cpus"]
```

Comment:
As @thekad mentioned, this is very clumsy. However, I am not sure how to make this better. Maybe you can loop through the keys of the config and use `setattr()`?

```python
# Not tested code
for key in config_file_data.keys():
    # TODO: Add error handling for when key not in marathon_app_result
    setattr(marathon_app_result, key, config_file_data[key])
```